### ISSUE TITLE
Game sequence re-runs when a user connects

### ISSUE THREAD

**Issue by mike055** [andyriffic/CNB#6]
## Scenario:
- Player A and B have submitted there selections
- Host clicks "play game"
- Game sequence runs
- Another player (or host) connects and the game sequence runs again

## Ideal:
- When a player connects it should not trigger the hosts game sequence to re-run
- Optionally, when a host connects and the game has already been run, do not re-run game sequence

## Ideas:
- Keep track of if the game has already been run on the server, and on the client only run the sequence if the game has never been run
  - After game has run, send message to the server saying it has been run?
- Work out what is changing in state/context that is causing the re-render and fix it

**Comment by andyriffic** [comment]
Yep, if we were to do it client side, then in `SocketsConnection.js` is where we receive game state from the server, then update local client state.

```js
    socket.on('GAME_VIEW', data => {
      // INSERT HERE -> check gameState (local) with data (server) and only call setGameState if something we care about changes
      setGameState(data);
    });
```
